IdentityServer/Startup.cs

```diff
@@ -237,16 +230,11 @@ private static void UpdateDatabase(IApplicationBuilder app, IWebHostEnvironment
                                                   .CreateScope();
             using IdentityDbContext context = serviceScope.ServiceProvider.GetService<IdentityDbContext>();
             context.Database.Migrate();
-            List<IdentityUser> identityUsers = TestUsers.GetDefaultIdentityUsers();
-            foreach(IdentityUser identityUser in identityUsers.Where(identityUser => !context.IdentityUser.Any(e => e.SubjectId == identityUser.SubjectId)))
+            if(!context.IdentityUser.Any())
             {
-                if(env.IsProduction())
-                {
-                    identityUser.Password = TestUsers.CreateTestUserPassword(identityUser.Username);
-                }
-                context.Add(identityUser);
+                context.IdentityUser.AddRange(TestUsers.GetDefaultIdentityUsers(env.IsProduction()));
```

Comment:
Something probably went wrong here while merging. Please check the file on develop: https://github.com/DigitalExcellence/dex-backend/blob/develop/IdentityServer/Startup.cs#L233-L252

Reply:
Make sure the seed file is also properly merged with develop